lib/app/frontend/Logger.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) 2017, Shopgate, Inc. All rights reserved.
+ *
+ * This source code is licensed under the Apache 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const { cyan, green, blue, red, bold } = require('chalk')
+
+let instance
+
+/**
+ * The Logger class hold static functions for logging usefull informations when running
+ * the Rapid Dev Server and the Webpack Dev Server.
+ * @type {Logger}
+ */
+class Logger {
```

Comment:
rename this to something like "LogHelper"
this should use the logger under lib/logger.js instead of console